lib/elements/satin_column.py

```diff
@@ -616,10 +692,10 @@ def _csp_to_satin(self, csp):
     @cache
     def center_line(self):
         # similar technique to do_center_walk()
-        center_walk, _ = self.plot_points_on_rails(self.zigzag_spacing, -100000)
+        center_walk, _ = self.plot_points_on_rails(self.zigzag_spacing, (0, 0), (-0.5, -0.5))
         return shgeo.LineString(center_walk)
 
-    def offset_points(self, pos1, pos2, offset_px):
+    def offset_points(self, pos1, pos2, offset_px, offset_prop):
```

Comment:
Could you please name this "offset_proportional"?  I know it's more to type but it's going to make this code much easier to understand.